README.md

```diff
@@ -68,15 +68,11 @@ export default defineComponent({
   plugins: [
     // ...your plugin
     compression(),
-    compression({ algorithm: 'brotliCompress', exclude: [/\.(br)$/, /\.(gz)$/], deleteOriginalAssets: true })
+    compression({ algorithm: 'brotliCompress' })
   ]
 })
 ```
 
-- You should set `exclude` for the latest compression plugin. ( Because in the full bundle process. Bundle chunk is shared. And
-  there are write (delete) operations on the bundle inside the plugin. So you should ignore the compressed chunk :) If you want delete
-  the original assets you also follow the way.
-

```

Comment:
The change here is wrong, As the follow explained below. setup two plugins and they will share the same rollup chunk instance. If user don't set `exclude` for the second plugin. the `.gz` asserts will be compressed again as  `.gz.br`.